applications/StructuralMechanicsApplication/tests/test_truss_adjoint_element_3d2n.py

```diff
@@ -0,0 +1,288 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import math
+
+def get_displacement_vector(mp,disp):
+    index=0
+    for node in mp.Nodes:
+        disp[index] = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X,0)
+        index = index + 1
```

Comment:
mp -> model_part

`index += 1`

even better would be 
```python
for i, node in enumerate(mp.Nodes):
    disp[i*3+0] = ...
    disp[i*3+1] = ...
    disp[i*3+2] = ...
```

Reply:
done